app/styles/components/export-tool.scss

```diff
@@ -0,0 +1,26 @@
+.content {
```

Comment:
Not trying to nitpick too hard, and I'm guilty of this too, but names like 'content' and 'component-container' scape me as CSS classes. 

Namespacing all of these with 'export-tool-' would go a long way for preventing future CSS collisions.